{"name": "Получение заказа неавторизованного пользователя", "status": "passed", "description": "Проверка: статус 401, сообщение с ошибкой.", "start": 1724955233402, "stop": 1724955233792, "uuid": "c7079dd9-9e13-4eb3-8051-c514c927022c", "historyId": "d5b374693486e05eab322f9924cb1832", "testCaseId": "d5b374693486e05eab322f9924cb1832", "fullName": "tests.test_order.TestOrder#test_get_orders_without_auth", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_order"}, {"name": "subSuite", "value": "TestOrder"}, {"name": "host", "value": "LAPTOP-74V5TFPG"}, {"name": "thread", "value": "8416-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_order"}]}